lua/starfall/libs_sv/nextbot.lua

```diff
@@ -0,0 +1,598 @@
+local registerprivilege = SF.Permissions.registerPrivilege
+local checkluatype = SF.CheckLuaType
+
+--- NextBot type
+-- @name NextBot
+-- @class type
+-- @server
+-- @libtbl nb_methods
+-- @libtbl nb_meta
+SF.RegisterType("NextBot", false, true, debug.getregistry().NextBot, "Entity")
+
+--- Library for spawning NextBots.
+-- @name nextbot
+-- @server
+-- @class library
+-- @libtbl nextbot_library
+SF.RegisterLibrary("nextbot")
+
+registerprivilege("nextbot.create", "Create nextbot", "Allows the user to create nextbots.")
+registerprivilege("nextbot.addContactCallback", "Add contact callback", "Allows the user to add a collision callback to the entity which is called every tick when touched.", {usergroups = {default = 1} } )
```

Comment:
This should have entities = {}. Probably want to move it to the other callback permissions too.